docs/sdk/dotnet-blazor-wasm/README.mdx

```diff
@@ -0,0 +1,247 @@
+---
+sidebar_label: .NET Core (Blazor WASM)
+---
+
+import ApiResourcesDescription from '../fragments/_api_resources_description.md';
+import AppNote from '../fragments/_app-note.mdx';
+import FurtherReadings from '../fragments/_further-readings.md';
+import ScopeClaimList from '../fragments/_scope-claim-list.md';
+
+# .NET Core: Integrate Logto with Blazor WebAssembly
+
+<AppNote type="Single page app" />
+
+## Get started
```

Comment:
The structure is not aligned with our new templates, @xiaohanyu could you please take a look?

Reply:
Yes this one is kind of tricky since it doesn't need our official SDK. I can submit a pull later to try my best for alignment.